conf/example.yaml

```diff
@@ -88,3 +88,7 @@ clustering_descriptor_selector:
       DELETE_COLUMN: 1
     column_mask_bitsets: null
     max_partition_size: 100
+
+# Default Row Visitor
+row_visitor:
+  default: {}
```

Comment:
this is a required config (without it it throws a NPE). It was set in the hardcoded runner, but not here in the YAML.